source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -67,6 +106,20 @@ Config::Config(
       pass_through_tlvs_.insert(0xFF & tlv_type);
     }
   }
+
+  for (const auto& version : proto_config.disallowed_versions()) {
+    switch (version) {
+    case ProxyProtocolConfig::V1:
+      allow_v1_ = false;
+      break;
+    case ProxyProtocolConfig::V2:
+      allow_v2_ = false;
+      break;
+    default:
+      throw EnvoyException(
+          absl::StrCat("Unknown proxy protocol version (enum int cast): ", version));
```

Comment:
We can use this https://github.com/envoyproxy/envoy/blob/ffcc257e16c9046b2fec7497a6bf9293d8ada286/source/common/common/assert.h#L309

Reply:
Thanks! Done